src/IMangrove.sol

```diff
@@ -62,118 +101,334 @@ interface IMangrove is HasMgvEvents {
     uint maxGasreqForFailingOffers
   ) external returns (uint takerGot, uint takerGave, uint bounty, uint fee);
 
+  ///@notice Performs a market order on a specified offer list taking offers up to a limit price.
```

Comment:
Maybe specify that the limit price is defined by a ratio of inbound/outbound volumes?

Eg
```suggestion
  ///@notice Performs a market order on a specified offer list taking offers up to a limit price defined by a ratio `inbound/outound` of volumes.
```

Reply:
resolved manually.